### PR TITLE
Fix resource samples in CSV

### PR DESCRIPTION
**Pull request by pmacik**
Signed-off-by: Pavel Macík <pavel.macik@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] All acceptance test scenarios included in the PR which verifies a bugfix or a requested feature reported by a non-member are tagged with `@external-feedback` tag.
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

File: config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: servicebinding.io/v1alpha3
```
Comment (sadlerap) [inline config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml:1]:
```suggestion
apiVersion: servicebinding.io/v1beta1
```

**Review by sadlerap — COMMENTED** [review]
Apart from one change, lgtm.

File: config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml (hunk 1 above)
Comment (pmacik) [reply config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml:1]: This is meant for the older `v1alpha3` version (see the file name). `v1beta1` is covered by the `spec_v1beta1_clusterworkloadresourcemapping.yaml`

Comment (pmacik) [reply config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml:1]: You see, when operator-sdk validates bundle, it complains if an example is missing for any CRD version owned by CSV. Which in our case includes both `v1alpha3` and `v1beta2` versions of the SPEC APIs.

Comment (pmacik) [reply config/samples/spec_v1alpha3_clusterworkloadresourcemapping.yaml:1]: That is why I included all of the missing ones.

**Review by sadlerap — APPROVED** [review]
/lgtm

**Pull request merged**